x-pack/platform/packages/shared/kbn-inference-cli/README.md

```diff
@@ -0,0 +1,40 @@
+# @kbn/inference-cli
+
+Exposes an Inference (plugin) API client for scripts, that mimicks the `chatComplete`
+and `output` APIs that are available on its start contract. It depends on the KibanaClient
+that is exposed from the `@kbn/kibana-api-cli` package. It automatically selects a
+connector if available. Usage:
+
+```ts
+import { createInferenceClient } from '@kbn/inference-cli';
+import { ToolingLog } from '@kbn/tooling-log';
+
+const log = new ToolingLog();
+const inferenceClient = await createInferenceClient({
+  log,
+  // pass in a signal that is triggered on teardown
+  signal: new AbortController().signal,
+});
+
+const response = await inferenceClient.output({
```

Comment:
There's no documentation on the `chatCompletion` endpoint. Can you add that, and also a description when to use one over the other.

I assume `chatComplete` allows me to pass in arbitrary functions whereas `output` is hardcoded to a single function (`structuredOutput`)